packages/client/src/Views/GameSetupNew.jsx

```diff
@@ -233,22 +232,11 @@ const AdminGameSetup = () => {
     }))
   }
 
-  const onCreateChannel = (buttonText) => {
+  const onCreateChannel = (id, createdChannel) => {
     if (channels.dirty) {
       dispatch(modalAction.open('unsavedChannel', 'create-new'))
     } else {
-      let id = `channel-${uniqid.time()}`
-      if (buttonText && (buttonText === CHANNEL_MAPPING || buttonText === CHANNEL_RFI_STATUS)) {
-        id = buttonText
-      }
-      dispatch(addNewChannel({ name: id, uniqid: id }))
-      dispatch(setSelectedChannel({ name: id, uniqid: id }))
-
-      const template = channelTemplate
-      template.name = id
-      template.uniqid = id
-
```

Comment:
@lilitkarapetyan - in this new way of doing it, where does `template.uniqid` get set ?

Reply:
This function was added up to this point but not used, the working one was in `packages/client/src/local/organisms/settings-channels/helpers/createChannel.ts`.
And everything works also only with the ability to save.

Reply:
Aah, great = thanks :-)